src/components/10-atoms/input-file/index.scss

```diff
@@ -1,3 +1,7 @@
+axa-input-file {
+  display: inline-block;
```

Comment:
Why do you add display? At my chrome everything works without it.

Reply:
Without it the component doesn't get selected properly & we could use it to allow 100% width.

<img width="224" alt="Screenshot 2020-07-21 at 08 49 07" src="https://user-images.githubusercontent.com/52913998/88022373-6c5a9a80-cb2f-11ea-8c05-93584cb544f9.png">

Reply:
Very nice!!! Please add this new functionality to the readme.